oci.go

```diff
@@ -302,3 +310,38 @@ func noNeedForOutput(detach bool, tty bool) bool {
 
 	return true
 }
+
+func getCgroupsDirPath() (string, error) {
+	if cgroupsDirPath != "" {
+		return cgroupsDirPath, nil
+	}
+
+	f, err := os.Open("/proc/self/mountinfo")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	var cgroupRootPath string
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		text := scanner.Text()
+		fields := strings.Split(text, " ")
+		index := strings.Index(text, " - ")
```

Comment:
Can you add a check on this ("`if index < 0`"). I'd also move the `fields :=` to the line after this one to avoid the `Split()` in case the `Index()` fails.